package.json

```diff
@@ -7,17 +7,21 @@
     "ajv": "^6.8.1",
     "readable-stream": "^3.1.1"
   },
+  "types": "index.d.ts",
   "devDependencies": {
+    "@types/node": "^11.11.4",
     "form-data": "^2.3.3",
     "pre-commit": "^1.2.2",
     "standard": "^12.0.0",
-    "tap": "^12.5.1"
+    "tap": "^12.5.1",
+    "tsd": "^0.7.0"
   },
   "scripts": {
     "test": "npm run lint && npm run unit",
     "lint": "standard",
     "unit": "tap test/test.js",
-    "coverage": "npm run unit -- --cov --coverage-report=html"
+    "coverage": "npm run unit -- --cov --coverage-report=html",
+    "test:tsd": "tsd"
```

Comment:
How does this work?

Reply:
Sorry this is incomplete. `tsd` is the typescript definition tester. All it needs is this command and then the `tsd: {}` options further down the file. It runs any file with `.test-d.ts` located in the `test` directory. I'm still working with it so I'll push the test file later